enum/anrede/anrede.go

```diff
@@ -8,11 +8,15 @@ type Anrede int
 
 const (
 	// HERR (male)
-	HERR             Anrede = iota + 1
-	FRAU                    // FRAU (female)
-	DIVERS                  // DIVERS (non-binary)
-	EHELEUTE                // EHELEUTE (married couple)
-	FIRMA                   // FIRMA (company)
-	WOHNGEMEINSCHAFT        // WOHNGEMEINSCHAFT is a share flat/WG
-	INDIVIDUELL             // INDIVIDUELL festgelegt
+	HERR                    Anrede = iota + 1
+	FRAU                           // FRAU (female)
+	DIVERS                         // DIVERS (non-binary) - Unofficial / Customer requirement, not part of the BO4E standard yet.
+	EHELEUTE                       // EHELEUTE (married couple)
+	FIRMA                          // FIRMA (company)
+	WOHNGEMEINSCHAFT               // WOHNGEMEINSCHAFT is a share flat/WG - Unofficial / Customer requirement, not part of the BO4E standard yet.
+	INDIVIDUELL                    // INDIVIDUELL festgelegt
+	FAMILIE                        // FAMILIE (family) - Unofficial / Customer requirement, not part of the BO4E standard yet.
+	ERBENGEMEINSCHAFT              // ERBENGEMEINSCHAFT (community of heirs, community of co-heirs, community of joint heirs) - Unofficial / Customer requirement, not part of the BO4E standard yet.
+	GRUNDSTUECKGEMEINSCHAFT        // GRUNDSTUECKGEMEINSCHAFT - Unofficial / Customer requirement, not part of the BO4E standard yet.
+	DR                             // DR (doctor) - Unofficial / Customer requirement, not part of the BO4E standard yet.
```

Comment:
I will check if we can add these to the official standard

Reply:
Habe ein Issue dafür im BO4E repo aufgemacht: https://github.com/bo4e/BO4E-python/issues/657